common/time.hpp

```diff
@@ -15,4 +15,12 @@ static inline Time timestamp() {
     return (Time)time.tv_sec * 1000000 + (Time)time.tv_usec;
 }
 
+static inline RJ::Time SecsToTimestamp(double secs) {
```

Comment:
Yeah that'd be good.  Just mention that it converts from a decimal number of seconds to an integer number of microseconds (a timestamp).  And vice versa for the other one.